### ISSUE TITLE
Question on the number of decoding sentences

### ISSUE THREAD

**Issue by cdjhz** [abisee/pointer-generator#45]
Hi @abisee, I've implemented your model and saw your test results. It seems that the number of decoding sentences is quite fixed. In my result, the number is roughly 3. The weird thing is that the program even replicates the second generated sentence to obtain the third sentence. 
Is there any constraint on the number of decoding sentences in the code? Or you just let the program to determine where to stop the whole summary, which means it actually leans to place comma and period properly.

**Comment by abisee** [comment]
> Or you just let the program to determine where to stop the whole summary, which means it actually leans to place comma and period properly.

Yes, that's how it's done. But you could easily change the code in `beam_search.py` to do something different e.g. produce a set number of sentences.
